packages/stark-core/src/user/entities/user.entity.ts

```diff
@@ -28,27 +28,27 @@ export class StarkUser implements StarkUserProfile, StarkUserSecurityProfile, St
 	@autoserialize
 	public lastName: string;
 
-	@ValidateIf(StarkValidationMethodsUtil.validateIfDefinedAndNotNull)
+	@ValidateIf(StarkValidationMethodsUtil.validateIfDefined)
```

Comment:
Why we now validate only for "undefined" values?  In fact we need to validate for "undefined" and "null" values because the config file (app-config.json) could have some values set to null...

The same applies for the other validations that were changed...